crates/governance/src/finalize_block.rs

```
@@ -324,6 +324,7 @@ where
         if vote.is_validator() {
             let vote_data = vote.data.clone();
 
+            #[allow(clippy::disallowed_methods)]
```

let's propagate the error with `?` instead in here